osu.Game.Tests/Visual/Playlists/TestScenePlaylistsMatchSettingsOverlay.cs

```diff
@@ -141,6 +141,18 @@ protected class TestRoomManager : IRoomManager
 
             public IBindableList<Room> Rooms => null;
 
+            public void AddOrUpdateRoom(Room room)
+            {
+            }
+
+            public void RemoveRoom(Room room)
+            {
+            }
+
+            public void ClearRooms()
+            {
+            }
```

Comment:
Should throw `NotImplementedException`, inline with the other irrelevant methods in this class.
```suggestion
            public void AddOrUpdateRoom(Room room) => throw new NotImplementedException();

            public void RemoveRoom(Room room) => throw new NotImplementedException();

            public void ClearRooms() => throw new NotImplementedException();
```